components/modal/modal.css

```diff
@@ -0,0 +1,41 @@
+@import url("/docs/main.css");
+
+.modal_box {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    display: none;
+    background-color: var(--white-color);
+}
+
+.modal {
+    width: 35vw;
+    height: 20vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    row-gap: 1rem;
+    border: 1px solid var(--primary-color);
+    background-color: var(--line-color);
+}
+
+.modal_active {
+    z-index: 1;
+    display: flex;
+    background-color: var(--);
```

Comment:
Add a variable name to `background-color`

Reply:
Done!